openpype/hosts/traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -0,0 +1,767 @@
+import os
+import re
+import csv
+import clique
+from copy import deepcopy, copy
+
+from openpype.client import get_asset_by_name
+from openpype.pipeline.create import get_subset_name
+from openpype.pipeline import CreatedInstance
+from openpype.lib import FileDef, BoolDef
+from openpype.lib.local_settings import get_openpype_username
+from openpype.lib.transcoding import (
+    VIDEO_EXTENSIONS, IMAGE_EXTENSIONS
+)
+
+from openpype.hosts.traypublisher.api.plugin import (
+    TrayPublishCreator
+)
+
+
+class IngestCSV(TrayPublishCreator):
+    """Editorial CSV creator class"""
+
+    icon = "fa.file"
+
+    label = "CSV Ingest"
+    family = "editorialcsv"
+    identifier = "io.openpype.creators.traypublisher.csv_ingest"
+
+    default_variants = ["Main"]
+
+    description = "Ingest products' data from CSV file"
+    detailed_description = """
+Ingest products' data from CSV file following column and representation
+configuration in project settings.
+"""
+
+    # Position batch creator after editorial creator
+    order = 10
+
+    def apply_settings(self, project_settings):
+        creator_settings = (
+            project_settings["traypublisher"]["create"]["IngestCSV"]
+        )
+        self.column_config = creator_settings["columns_config"]
+        self.representation_config = creator_settings["representations_config"]
+
```

Comment:
I think this is not needed and settings should be applied automatically (need to be validated).
```suggestion
```